### ISSUE TITLE
[error]One of your files includes a space.

### ISSUE THREAD

**Issue by pforrence-inquirer** [jitterbit/get-changed-files#12]
`##[error]One of your files includes a space. Consider using a different output format or removing spaces from your filenames. Please submit an issue on this action's GitHub repo.`

When a filename contains a space, the setup fails.

**Comment by sean-krail** [comment]
Hey @pforrence-inquirer, try using the csv or json output formats if your file names contains spaces. There's examples in the README on how to do that.

**Comment by pforrence-inquirer** [comment]
@sean-krail thank you! I'll give it a go

**Comment by pbek** [comment]
> try using the csv or json output formats if your file names contain spaces. There's examples in the README on how to do that.

Unfortunately that doesn't work under macOS, #15 😞

**Comment by pforrence-inquirer** [comment]
It worked for me. If you're doing it in a github action I believe you can specify a unix/linux box

`# A workflow run is made up of one or more jobs that can run sequentially or in parallel
jobs:
  # This workflow contains a single job called "build"
  build:
    # The type of runner that the job will run on
    runs-on: ubuntu-latest`

**Comment by pbek** [comment]
> It worked for me. If you're doing it in a github action I believe you can specify a unix/linux box

yes, it works under Windows (when using bash) and Linux, but not under macOS (but I found a workaround in #15)
